ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh

```diff
@@ -16,6 +16,23 @@ tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/assisted.tar.gz"
 # Prepare configuration and run
 scp "${SSHOPTS[@]}" "${CLUSTER_PROFILE_DIR}/pull-secret" "root@${IP}:pull-secret"
 
+if [ "${ENVIRONMENT}" != "local" ]; then
+  if [ "${ENVIRONMENT}" = "production" ]; then
+    remote_service_url="https://api.openshift.com"
+    pull_secret_file=/var/run/assisted-installer-bot/prod-pull-secret
+  else
+    echo "Unknown environment ${ENVIRONMENT}"
+    exit 1
+  fi
+
+  scp "${SSHOPTS[@]}" /var/run/assisted-installer-bot/offline-token "root@${IP}:offline-token"
+  scp "${SSHOPTS[@]}" "${pull_secret_file}" "root@${IP}:pull-secret"
+  echo "export REMOTE_SERVICE_URL=${remote_service_url}" >> "${SHARED_DIR}/assisted-additional-config"
+  echo "export NO_MINIKUBE=true" >> "${SHARED_DIR}/assisted-additional-config"
+  echo "export MAKEFILE_TARGET='create_full_environment test_parallel'" >> "${SHARED_DIR}/assisted-additional-config"
+  echo "export WORKER_DISK=128849018880" >> "${SHARED_DIR}/assisted-additional-config"
```

Comment:
This needs explaining, I have no idea what that is

Reply:
Oh is that like disk size

Reply:
Can we put the number in a seperate var called 120GiB or something

Reply:
done